app/OllamaService.php

```diff
@@ -3,31 +3,43 @@
 namespace App;
```

Comment:
Here are my observations:

- The `Exception` class is commented out in the class definition. Remove the `-use Exception;` line to fix this.

No other issues were found with the provided code snippet.